com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDRenderPipelineUI.cs

```diff
@@ -356,7 +356,11 @@ static void Drawer_SectionShadows(SerializedHDRenderPipelineAsset serialized, Ed
 
             if (!serialized.renderPipelineSettings.supportedLitShaderMode.hasMultipleDifferentValues)
             {
-                EditorGUILayout.PropertyField(serialized.renderPipelineSettings.hdShadowInitParams.shadowFilteringQuality, Styles.filteringQuality);
+                // TEMP: HDShadowFilteringQuality.VeryHigh - This filtering mode is not ready so disabling in UI
+                // To re-enable remove the wo following light and re-enable the third one
+                int value = EditorGUILayout.IntPopup(Styles.filteringQuality, serialized.renderPipelineSettings.hdShadowInitParams.shadowFilteringQuality.enumValueIndex, Styles.shadowFilteringNames, Styles.shadowFilteringValue);
+                serialized.renderPipelineSettings.hdShadowInitParams.shadowFilteringQuality.enumValueIndex = value;
+                //EditorGUILayout.PropertyField(serialized.renderPipelineSettings.hdShadowInitParams.shadowFilteringQuality, Styles.filteringQuality);
```

Comment:
If you want to disable the entry maybe is better to do it this way:

![image](https://user-images.githubusercontent.com/76204843/133213590-2fa5a994-b981-40c5-8a0d-7ea4c5211cc9.png)

```suggestion
serialized.renderPipelineSettings.hdShadowInitParams.shadowFilteringQuality.intValue = (int)(HDShadowFilteringQuality)EditorGUILayout.EnumPopup(
                    Styles.filteringQuality,
                    (HDShadowFilteringQuality)serialized.renderPipelineSettings.hdShadowInitParams.shadowFilteringQuality.intValue,
                        e => (HDShadowFilteringQuality)e != HDShadowFilteringQuality.VeryHigh, /* TEMP: HDShadowFilteringQuality.VeryHigh - This filtering mode is not ready so disabling in UI */
                    false);
```

It will still be visible, but users won't be able to select it from the UI, Also you do not need the `GUIContents`

Reply:
Thanks for the suggestion and it is what I wanted to do initially but don't know how, so good. But it is preferable to hide it fully, otherwise we will get various question about how to enable it. i.e users will think there is a settings somewhere to enable it whereas the feature is not implemented at all...